### PR TITLE
add the code to analize

### PR DESCRIPTION
**Pull request by sjdev2212**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header id="header">
```
Comment (sjdev2212) [inline index.html:9]: please try to use class instead of id

File: index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      </div>
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#features">Features</a></li>
+          <li><a class="nav-link" href="#how-it-works">How it Works</a></li>
+          <li><a class="nav-link" href="#pricing">Pricing</a></li>
+        </ul>
+      </nav>
+    </header>
+    <setion>
```
Comment (sjdev2212) [inline index.html:25]: please check the spelling  - should be "section"

File: style.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  }
+  
+  @media (max-width: 600px) {
+    header {
+      flex-wrap: wrap;
+    }
+    #price {
+      flex-direction: column;
+    }
+  }
```
Comment (sjdev2212) [inline style.css:52]: Please try to use linters such as Stylelint to improve the code

**Review by sjdev2212 — COMMENTED** [review]
#Changes required♻

Please check the suggested changes below.
Submit for a re-review when you are done.😀  Cheers!!

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @sj1978  🔥 

Good review! Now you can continue to the next milestone!👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**